<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_01</class>
 <widget class="QMainWindow" name="MainWindow_01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="BackGround">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#BackGround{
	border-image: url(:/new/prefix1/newback01.jpg);

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="StartGame">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>490</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pic.qrc">
       <normaloff>:/picture/visitor.png</normaloff>:/picture/visitor.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="help">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>300</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/bottons.qrc);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pic.qrc">
       <normaloff>:/picture/help.png</normaloff>:/picture/help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ourteam">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>580</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pic.qrc">
       <normaloff>:/picture/ourteam.png</normaloff>:/picture/ourteam.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>390</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="pic.qrc">
       <normaloff>:/picture/login.png</normaloff>:/picture/login.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>230</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>白方：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="user_white">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>黑方：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="user_black">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="Broad">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>9x9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11x11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13x13</string>
      </property>
     </item>
    </widget>
    <zorder>help</zorder>
    <zorder>StartGame</zorder>
    <zorder>ourteam</zorder>
    <zorder>login</zorder>
    <zorder>widget_3</zorder>
    <zorder>Broad</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
